### ISSUE TITLE
Docker hub tags

### ISSUE THREAD

**Issue by gamerlv** [elementary/docker#42]
Since #32 was merged the tags on docker hub have not been updated. I've opened #41 to update the readme here, but that does not change anything for people using the hub directly.

Should the description there be updated to point users to github and/or remove the tags to make it painfully obvious.  
Removing the tags probably will break some builds, as some tags are still pulled. At time of writing `latest` was last pulled only 3 hours ago.

**Comment by balupton** [comment]
Yeah, I'd like to test https://github.com/bevry/dorothy against Elementary, especially for arm, however https://hub.docker.com/r/elementary/docker last updates were 3 years ago
